src/shared/utils.js

```diff
@@ -67,28 +69,37 @@ export const Utils = {
    * @param {string} url - URL from which to retrieve the domain name.
    * @returns {string} - The domain name, or the url if a valid (within the context of the extension) domain name is not derived.
    */
-  getTopLevelDomain(url) {
-    url = this.stripPrefixesFromUrl(url);
-
-    try {
-      // Create a URL object from the input URL
-      let parsedUrl = new URL(url);
+getDomainName(url) {
```

Comment:
NITPICKING: this is technically now an [eTLD ](https://developer.mozilla.org/de/docs/Glossary/eTLD)("effective top-level domain").

Reply:
This is great nitpicking but I'm not sure `eTLD` is totally correct here! I actually think `eTLD+1` may be the _most_ correct term here or possibly `registrable domain` per [the URL spec](https://url.spec.whatwg.org/#host-registrable-domain) but this is iffy too.

Reply:
I'm going to merge this, but I'm totally happy to revisit the nomenclature here.